### PR TITLE
Removes a shit clown name and adds a better clown name instead

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Removes a shit clown name and adds a better clown name instead

## Why It's Good For The Game

Disgraced nazi artist has been replaced with a professional clown who's an expert in cocaine. Someone brought up we didn't have this in the clown name pool.

## Changelog
:cl:
spellcheck: Removes a shit clown name and adds a better clown name instead
/:cl:

### CODE REVIEW

File: strings/names/clown.txt [hunk 1]
```diff
@@ -118,7 +118,7 @@ Unimaginable Nut
 Valid Vincent
 Weather Report
 Widderwise
-Yanye Kest
+Doctor Rockso
```
Comment (MrMelbert) [inline strings/names/clown.txt:121]: The only clown here is you for not putting this in ABC order

**Comment by iwishforducks** [comment]
yanye kest is such an unbelievably bad name past all the controversy. this made me laugh.

**Comment by DATA-xPUNGED** [comment]
Nevermind the PR change, i've got a great clown idea with the name Yesterday's Beef

**Comment by nianjiilical** [comment]
if you're going to use gbp to change clown names you ought to make a players club thread to gather suggestions and add more than one, it might be fun

**Comment by Iamgoofball** [comment]
make one then, also this PR gives me GBP

**Comment by Dorsisdwarf** [comment]
I didnt even realise this was on the list, nice catch

**Pull request merged**